agent/task.go

```diff
@@ -91,6 +91,22 @@ func (a *Agent) startTask(ctx context.Context, tc *taskContext, complete chan<-
 	)
 	tc.statsCollector.logStats(innerCtx, tc.taskConfig.Expansions)
 
+	tc.systemMetricsCollector, err = NewSystemMetricsCollector(
```

Comment:
Not a code change, but this will have to be tested in staging before committing because it will change the agent.

Also, you have to update the agent version date in `config.go`.